Core/include/Acts/Seeding/SeedFinderGbts.hpp

```diff
@@ -51,47 +51,47 @@ class SeedFinderFTF {
   //   using tree_t = KDTree<NDims, internal_sp_t *, ActsScalar, std::array, 4>;
 
   // constructors
-  SeedFinderFTF(const SeedFinderFTFConfig<external_spacepoint_t> &config,
-                const TrigFTF_GNN_Geometry<external_spacepoint_t> &GNNgeo);
+  SeedFinderGbts(const SeedFinderGbtsConfig<external_spacepoint_t> &config,
+                 const GbtsGeometry<external_spacepoint_t> &Gbtsgeo);
```

Comment:
```suggestion
  SeedFinderGbts(const SeedFinderGbtsConfig<external_spacepoint_t> &config,
                 const GbtsGeometry<external_spacepoint_t> &gbtsGeo);
```

For variable names we should start with lower case. (Also some occurrences below)